.appveyor.yml

```diff
@@ -39,7 +39,7 @@ install:
   - CD ..
 
   # install and build go-libp2p-daemon
-  - bash scripts/build_p2pd.sh p2pdCache v0.2.4
+  - bash scripts/build_p2pd.sh p2pdCache update
```

Comment:
what's `update`? thought we tested against a particular branch to avoid upstream-caused breakage?

Reply:
I wish, but the go daemon last I checked did not support 1.1 officially, this update was a branch they had "WIP"

Reply:
Looks like they finally merged! will fix it!

Reply:
fixed